architecture/core_concepts/pods_and_services.adoc

```diff
@@ -271,3 +271,31 @@ A service or replication controller that is defined to use pods with the
 The
 https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/user-guide/labels.md[Kubernetes
 documentation] has more information on labels.
+
+[[endpoints]]
+
+== Endpoints
+
+The servers that back a service are called its endpoints, and are
+specified by an object of type *Endpoints* with the same name as the
+service. When a service is backed by pods, those pods will normally be
+specified by a label selector in the service specification, and
+{product-title} will automatically create the Endpoints object
+pointing to those pods.
+
+In some cases you may want to create a service but have it be backed
+by external hosts rather than by pods in the {product-title} cluster.
+In this case, you can leave out the `*selector*` field in the service,
+and
+xref:../../dev_guide/integrating_external_services.adoc#dev-guide-integrating-external-services[create
+the Endpoints object manually].
+
+Note that {product-title} will not let most users manually create an
+Endpoints object that points to pod or service IPs. (Since these IPs
+are normally dynamically assigned, it doesn't make much sense to try
+to track them manually anyway.) Only
+xref:../additional_concepts/authorization.adoc#roles[cluster admins]
+or other users with
+xref:../additional_concepts/authorization.adoc#evaluating-authorization[permission
+to `create` resources under `endpoints/restricted`] can create such
+Endpoint objects.
```

Comment:
do we document the master config options to change which IP ranges are considered restricted?